### PR TITLE
adding index.html, products.html and images

### PR DESCRIPTION
**Pull request by nathantbaker**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      <img src="images/hero.jpg" alt="megaDesk homepage image">
+    </div>
+
+    <div id="content">
+
+      <h1>If it's not megaDesk, it's a foot stool.</h1>
+
+      <article class="feature">
+        <h2>Space for 20+ Monitors</h2>
+        <p>You are important and your deak size should scream "I'm a big deal." Have enough space for all the monitors you need, plus 2 mini fridges, and 5 pets.</p>
```
Comment (sales-ls21) [inline index.html]: minor spelling edit : desk vice deak

File: products.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        <p class="description">Now you don't have to go outside for zen.</p>
+        <p class="price">$231</p>
+        <a class="button" href="#">Add to Cart</a>
+      </article>
+
+      <!-- Product 5 -->
+      <article class="card">
+        <h2>Lie Down Desk</h2>
+        <img src="images/lie-down-desk.jpg" alt="Lie Down Desk">
+        <p class="description">Don't liking being normal? This will help.</p>
```
Comment (sales-ls21) [inline products.html]: spelling: Don't LIKE being normal?
vice: "don't liking"

File: products.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        <h2>Tie Fighter Desk</h2>
+        <img src="images/tie-figher-desk.jpg" alt="Tie Fighter Desk">
+        <p class="description">It's Star Wars related so you have to buy it.</p>
+        <p class="price">$8,399</p>
+        <a class="button" href="#">Add to Cart</a>
+      </article>
+
+      <!-- Product 8 -->
+      <article class="card">
+        <h2>Toliet Desk</h2>
```
Comment (sales-ls21) [inline products.html]: spelling: Toilet

**Review by sales-ls21 — COMMENTED** [review]
Everything content wise looks good. Even when I tested the CSS. There are just a few minor spelling edits.

**Comment by nathantbaker** [comment]
changes made, the toilet typo also affected an image src, and I updated the src html and the file name.

**Pull request merged**